templates/cms/media.html

```diff
@@ -1,13 +1,13 @@
 {% extends "base.html" %}
 {% load static %}
 
-{% block headtitle %}{{media_object.title}} - {{PORTAL_NAME}}{% endblock headtitle %}
+{% block headtitle %}{{media_object.title|add:" - "|add:PORTAL_NAME|truncatechars:64}}{% endblock headtitle %}

```

Comment:
I think 
`{{media_object.title|truncatechars:5}} - {{PORTAL_NAME}}` is what you want here, if you want to truncate the title and keep the portal name at the end

Reply:
In this case, I figured it was more important to have the media title in there than to always have the portal name in there, which is why I truncated the concatenation instead.